<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDs4</class>
 <widget class="QWidget" name="IDs4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 4 TID/SID</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxShinyPID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Shiny PID</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelShinyPIDPID">
        <property name="text">
         <string>PID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TextBox" name="textBoxShinyPIDPID">
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxShinyPIDSearchTID">
        <property name="text">
         <string>Search for TID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelShinyPIDTID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TextBox" name="textBoxShinyPIDTID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelShinyPIDYear">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="TextBox" name="textBoxShinyPIDYear">
        <property name="text">
         <string notr="true">2000</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelShinyPIDMinDelay">
        <property name="text">
         <string>Min Delay</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="TextBox" name="textBoxShinyPIDMinDelay">
        <property name="text">
         <string notr="true">5000</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelShinyPIDMaxDelay">
        <property name="text">
         <string>Max Delay</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="TextBox" name="textBoxShinyPIDMaxDelay">
        <property name="text">
         <string notr="true">6000</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxShinyPIDInfiniteSearch">
        <property name="text">
         <string>Infinite Search</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonShinyPIDSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonShinyPIDCancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxTIDSID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>TID / SID</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTIDSIDTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TextBox" name="textBoxTIDSIDTID">
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxTIDSIDSearchSID">
        <property name="text">
         <string>Search for SID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTIDSIDSID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TextBox" name="textBoxTIDSIDSID">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelTIDSIDYear">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="TextBox" name="textBoxTIDSIDYear">
        <property name="text">
         <string notr="true">2000</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelTIDSIDMinDelay">
        <property name="text">
         <string>Min Delay</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="TextBox" name="textBoxTIDSIDMinDelay">
        <property name="text">
         <string notr="true">5000</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelTIDSIDMaxDelay">
        <property name="text">
         <string>Max Delay</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="TextBox" name="textBoxTIDSIDMaxDelay">
        <property name="text">
         <string notr="true">6000</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxTIDSIDInfiniteSearch">
        <property name="text">
         <string>Infinite Search</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonTIDSIDSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonTIDSIDCancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="progressBarShinyPID">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QProgressBar" name="progressBarTIDSID">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="TableView" name="tableViewShinyPID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="TableView" name="tableViewTIDSID">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="TableView" name="tableViewSeedFinder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBoxSeedFinder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Seed Finder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSeedFinderTID">
        <property name="text">
         <string>TID Obtained</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="TextBox" name="textBoxSeedFinderTID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSeedFinderDateTime">
        <property name="text">
         <string>Date/Time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="DateTimeEdit" name="dateTimeEdit">
        <property name="maximumDate">
         <date>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="minimumDate">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSeedFinderMinDelay">
        <property name="text">
         <string>Min Delay</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="TextBox" name="textBoxSeedFinderMinDelay">
        <property name="text">
         <string notr="true">5000</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelSeedFinderMaxDelay">
        <property name="text">
         <string>Max Delay</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="TextBox" name="textBoxSeedFinderMaxDelay">
        <property name="text">
         <string notr="true">6000</string>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonSeedFinderSearch">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>DateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>Forms/Controls/DateTimeEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>textBoxShinyPIDPID</tabstop>
  <tabstop>checkBoxShinyPIDSearchTID</tabstop>
  <tabstop>textBoxShinyPIDTID</tabstop>
  <tabstop>textBoxShinyPIDYear</tabstop>
  <tabstop>textBoxShinyPIDMinDelay</tabstop>
  <tabstop>textBoxShinyPIDMaxDelay</tabstop>
  <tabstop>checkBoxShinyPIDInfiniteSearch</tabstop>
  <tabstop>pushButtonShinyPIDSearch</tabstop>
  <tabstop>pushButtonShinyPIDCancel</tabstop>
  <tabstop>tableViewShinyPID</tabstop>
  <tabstop>textBoxTIDSIDTID</tabstop>
  <tabstop>checkBoxTIDSIDSearchSID</tabstop>
  <tabstop>textBoxTIDSIDSID</tabstop>
  <tabstop>textBoxTIDSIDYear</tabstop>
  <tabstop>textBoxTIDSIDMinDelay</tabstop>
  <tabstop>textBoxTIDSIDMaxDelay</tabstop>
  <tabstop>checkBoxTIDSIDInfiniteSearch</tabstop>
  <tabstop>pushButtonTIDSIDSearch</tabstop>
  <tabstop>pushButtonTIDSIDCancel</tabstop>
  <tabstop>tableViewTIDSID</tabstop>
  <tabstop>textBoxSeedFinderTID</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>textBoxSeedFinderMinDelay</tabstop>
  <tabstop>textBoxSeedFinderMaxDelay</tabstop>
  <tabstop>pushButtonSeedFinderSearch</tabstop>
  <tabstop>tableViewSeedFinder</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxShinyPIDSearchTID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxShinyPIDTID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxShinyPIDInfiniteSearch</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxShinyPIDMaxDelay</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTIDSIDSearchSID</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxTIDSIDSID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTIDSIDInfiniteSearch</sender>
   <signal>clicked(bool)</signal>
   <receiver>textBoxTIDSIDMaxDelay</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
